src/hooks/useBreakpointValue.ts

```diff
@@ -0,0 +1,3 @@
+import { useBreakpointValue } from '@chakra-ui/react';
```

Comment:
careful when using that hook. Is JS dependent so you'll get layout shifts. When it's above the fold, we usually render 2 variants and toggle visibility with CSS